### DISCUSSION TITLE
Case insensitive _contains in PostgreSQL

### DISCUSSION THREAD

**Discussion by HalikovStudioIT** [directus/directus#11552] [Archive]
How to make case insensitive filter in query _contains in PostgreSQL?

**Comment by azrikahar** [comment]
Currently filter operators like `_contains` does not support case insensitive yet, but there is an existing discussion for this: https://github.com/directus/directus/discussions/2981.

In the meantime, you can try to use `search` instead of `filter` which does case insensitive lookups: https://docs.directus.io/reference/query/#search. However as the docs outlined, it will search through all fields (except relational ones) so do take note on that.
